.github/ISSUE_TEMPLATE/report_documentation_issue.yml

```diff
@@ -0,0 +1,80 @@
+name: ❓ Report documentation issue
+description: Report problems and issues with Dagster documentation.
+labels: ["documentation", "content-gap"]
+body:
+  - type: markdown
+    attributes:
+      value: >
+        <p align="center">
+          <a target="_blank" href="https://dagster.io">
+            <picture>
+              <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/dagster-io/dagster/rl/06-28-docs_use_dagster_brand_colors_for_README/.github/dagster-logo-dark.svg">
+              <img alt="dagster logo" src="https://raw.githubusercontent.com/dagster-io/dagster/rl/06-28-docs_use_dagster_brand_colors_for_README/.github/dagster-logo-light.svg" align="left" width="auto" height="120">
+            </picture>
+          </a>
+        </p>
+  - type: markdown
+    attributes:
+      value: >
+        Thank you for contributing to improve Dagster.
+        If you have a change ready to submit, you do not need to create an issue.
+        Instead, you can open a [pull request](https://github.com/dagster-io/dagster/pulls).
+  - type: markdown
+    attributes:
+      value: >
+        Writer's Guide: [Docs README](https://github.com/dagster-io/dagster/tree/master/docs)
+  - type: textarea
+    attributes:
+      label: What content is missing
+      description: Help us to understand your request in context.
+      placeholder: Please provide a brief description of the missing content.
+    validations:
+      required: true
+  - type: dropdown
+    attributes:
+      label: Type of content
+      description: What type would the content most likely belong to?
+      multiple: true
+      options:
+        - "Tutorial"
+        - "Main Concept: 'What is X in Dagster' where X is a Dagster concept."
+        - "Integration Guide: 'How to use X in Dagster' where X is a 3rd-party package/tool, e.g. dbt, Spark, etc"
+        - "Best Practices Guide: 'How to do X in Dagster' where X is usually a use case and can be addressed by incorporating multiple Dagster concepts at a time."
+        - "Deployment Guide: 'How to deploy Dagster to X', e.g. Docker, K8S, GCP, etc"
+        - "API Reference: e.g. docstring"
+        - "Other"
```

Comment:
I'm not sure we need to keep this question. I'd rather make it easier for users to quickly submit issues and leave it up to us to decide where it goes/what it is than they have to think too much about it. Most issues I've looked at also didn't bother to fill this out (though I know it's marked as required on this form).

@yuhan @sryza - thoughts on scrapping this? Is there value in keeping it?